### PR TITLE
feat: temporary UI

### PR DESCRIPTION
**Pull request by alanshaw**
Mostly copied from the nft.storage website and bits trimmed out.

This is COMPLETELY disposable and only exists temporarily to try things out.

Screenshot 2021-07-03 at 11 37 43

### CODE REVIEW

**Comment by olizilla** [comment]
pls to resolve conflicts @alanshaw

File: packages/website/next-env.d.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/// <reference types="next" />
```
Comment (olizilla) [inline packages/website/next-env.d.ts:1]: what is?

Comment (alanshaw) [reply packages/website/next-env.d.ts:1]: IDK next.js make it

File: packages/website/package.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        }
+      ],
+      "react/display-name": 1
+    },
+    "settings": {
+      "react": {
+        "version": "detect"
+      }
+    }
+  },
```
Comment (olizilla) [inline packages/website/package.json:75]: note to self: we should unify this with the prettier config at the root of the project

File: packages/website/styles/global.css [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+  .table-responsive {
+    display: block;
+    width: 100%;
+    overflow-x: auto;
+  }
```
Comment (olizilla) [inline packages/website/styles/global.css:11]: _it's the smell_

**Review by olizilla — APPROVED** [review]

File: packages/website/styles/global.css (hunk 3 above)
Comment (alanshaw) [reply packages/website/styles/global.css:11]: 😉 https://github.com/ipfs-shipyard/nft.storage/pull/61#discussion_r618455074

**Pull request merged**